ci/jenkins/jobs/macros.yaml

```diff
@@ -254,7 +254,27 @@
           return_code=$?
           set -ex
           ./ci/kind/kind-setup.sh destroy "${{FULL_CLUSTER_NAME}}"
-          exit $return_code 
+          exit $return_code
+                   
+- builder:
+    name: builder-kind-flexible-ipam-e2e
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -ex
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          KIND_TIMEOUT=135
+          FULL_CLUSTER_NAME='{kind_cluster_name}'-"${{BUILD_NUMBER}}"
+          ./ci/kind/kind-setup.sh destroy --all --until ${{KIND_TIMEOUT}}
+          ./ci/kind/kind-install.sh 
+          ./ci/kind/kind-setup.sh --antrea-cni --flexible-ipam create "${{FULL_CLUSTER_NAME}}" --ipam-vlan-subnets '192.168.241.1/24,192.168.242.1/24' --ipam-vlan-ids '11,12'
```

Comment:
let's use variables instead of using constant cidrs for vlansubnets and vlanid like 192.168.241.1/24,192.168.242.1/24